pkg/gen/awaitComments.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gen
+
+import (
+	"github.com/pulumi/pulumi-kubernetes/pkg/kinds"
+)
+
+const DeploymentAwaitComment = `Pulumi uses "await logic" to determine if a Deployment is ready.
```

Comment:
I don’t think we say “Pulumi used” here. We say something more like “this resource waits until deployment is complete before registering that the resource succeeded to create/update and populating output properties from the current state of the resource. The following conditions are used to determine whether the resource creation has succeeded or failed”.